src/lib/matchStatistics.ts

```diff
@@ -17,6 +18,15 @@ class MatchStatistics {
     );
   }
 
+  /**
+   * Gets all matches sorted by most recently registered match.
+   *
+   * @returns matches lol?
+   */
+  static getMatches() {
```

Comment:
Remove this, this is why the test are failing, you've essentially introduced a dependency on the database by referencing a query in here.. to resolve the query it needs to resolve the db and so forth. It also isn't used.

Reply:
🇩🇪